src/Stripe.net/Services/Invoices/InvoiceCreateOptions.cs

```diff
@@ -38,6 +38,9 @@ public class InvoiceCreateOptions : BaseOptions
         [JsonProperty("days_until_due")]
         public long? DaysUntilDue { get; set; }
 
+        [JsonProperty("default_source")]
+        public string DefaultSource { get; set; }
+
```

Comment:
If `AutoAdvance` belongs in `InvoiceParams` in Go, doesn't it belong in either create or update here too?

Reply:
Awesome, nice catch. I totally missed those 2 when I did the PR. Fixed, PTAL